public/images/check-mark-icon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const DoubleArrowsSelectIcon = () => (
```

Comment:
Is this meant to be `CheckMarkIcon`?

Reply:
Yep! Thank you for catching my bad copy-paste! Should be fixed by the latest commit